operations/cloud-maintenance/openshift/template.yaml

```diff
@@ -103,3 +176,17 @@ objects:
               aws s3 rm ${S3_CLEANER_EXTRA_PARAMS} "s3://${AWS_S3_BUCKET}/${cluster_id}/logs/${host_id}/boot_logs.tar.gz"
           done
       done
+    check_bootlogs: |
+      #!/bin/bash -e
+      uuid_pattern="[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/"
+      # detect UUID-like directories to loop through clusters
+      for cluster_id in $(aws s3 ls s3://${AWS_S3_BUCKET} | grep -E "${uuid_pattern}" | awk '{ print $2 }' | sed 's/\///g'); do
```

Comment:
btw, I see there's an option to do ``aws s3 <command> --include "*.jpg"`` for example, and ``--output json|text``, so at least some of the grep/awk/sed shouldn't be needed

Reply:
either way, it's just a nit. so approving regardless

Reply:
Nice, we can add it if we need another iteration